### DISCUSSION TITLE
Is there a way to connect the MonAI label to the study data source (if the data is hosted on S3 or on network storage)?

### DISCUSSION THREAD

**Discussion by hiq-ayeddula** [Project-MONAI/MONAILabel#1644] [Q&A]
Is there a way to connect the MonAI label to the study data source  (if the data is hosted on S3 or on network storage)

**Comment by diazandr3s** [comment]
Hi @hiq-ayeddula,

[MONAI Label supports DICOMWeb](https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/datastore/dicom.py). If you have the dataset in AWS, I'd recommend checking the solutions they have to serve images via DICOMWeb API.

Hope this helps,
